src/styles/fonts.css

```diff
@@ -0,0 +1,17 @@
+@font-face {
+  font-family: Roboto;
+  src: url(../fonts/Roboto-Regular.woff);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(../fonts/Roboto-Medium.woff);
+  font-weight: 500;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(../fonts/Roboto-Bold.woff);
+  font-weight: 700;
+}
```

Comment:
Why not google fonts? It will be much easier, and also there will be no need to upload font files